src/types/index.ts

```diff
@@ -0,0 +1,11 @@
+export type Severity = 'info' | 'warning' | 'error'
+export type ThemeType = 'light' | 'dark'
+export type Position = 'top-start' | 'top-end' | 'bottom-start' | 'bottom-end'
+
+export interface Props {
```

Comment:
"Props" is a too generic name.

Reply:
This is to be generic even though it is used in several places

Reply:
rename for "CommomProps" instead